### DISCUSSION TITLE
Hybrid application checklist experiments:

### DISCUSSION THREAD

**Discussion by commjoen** [OWASP/mastg#1971] [Ideas]
This issue covers delta analysis of the 3 most important hybrid platforms:

- [ ] React native: https://drive.google.com/open?id=1P5FZ_Bup5eSPOmkePZA8cIpKGOKvngkN
- [ ] Xamarin: https://drive.google.com/open?id=1UL1yLRREJwXfe0HlrcX-IuvPYQM7lTtG
- [ ] Flutter: https://drive.google.com/open?id=1wHK3VI1cU1xmYrCu9yb5OHKUEeLIPSkC

**Comment by commjoen** [comment]
Work is progressing on the xamarin sheet thanks to spirit. The react native sheet is slowly receiving updates..  flutter is getting no progress...

**Comment by righettod** [comment]
Based on this [talk](https://www.youtube.com/watch?v=jrnxYewex5w), perhaps sub area in a test case can be created.

Example for Android:

**Idea 1**

```markdown
# Testing Custom Certificate Stores and Certificate Pinning (MSTG-NETWORK-4)
## Native 
Technical instruction for native
## Xamarin
Technical instruction for xamarin
## Flutter
Technical instruction for flutter
...
```

**Idea 2**

Dedicated MD files, by framework, containing the technical instruction for each check point of the MSTG.
